openfl-workspace/flower-app-pytorch/plan/plan.yaml

```diff
@@ -22,7 +22,7 @@ connector :
     flwr_run_params :
       flwr_app_name : "app-pytorch"
       federation_name : "local-poc"
-      patch: True
+      SGX_enabled: False
```

Comment:
Nice! It's much more self-explanatory now than the somewhat crypting `patch` flag.